docs/ff-concepts/navigation-routing/bottom-sheet.md

```diff
@@ -0,0 +1,158 @@
+---
+title: Bottom Sheet
+slug: /concepts/navigation/bottom-sheet
+sidebar_position: 7
+tags: [Navigation, Actions]
+keywords: [FlutterFlow, Bottom Sheet, Navigation]
+---
+# Bottom Sheet
+A Bottom Sheet is an alternative to a menu or a dialog. It opens from bottom to top and can be dismissed by swiping it from top to bottom. When it opens, it prevents the user from interacting with the rest of the app. 
+
+You can use the bottom sheet when you want to perform a small action without creating a separate screen.
+
+<div style={{
+    position: 'relative',
+    paddingBottom: 'calc(56.67989417989418% + 41px)', // Keeps the aspect ratio and additional padding
+    height: 0,
+    width: '100%'}}>
+    <iframe 
+        src="https://demo.arcade.software/4GbJ3ujPrnvX6MwYiGdY?embed&show_copy_link=true"
+        title=""
+        style={{
+            position: 'absolute',
+            top: 0,
+            left: 0,
+            width: '100%',
+            height: '100%',
+            colorScheme: 'light'
+        }}
+        frameborder="0"
+        loading="lazy"
+        webkitAllowFullScreen
+        mozAllowFullScreen
+        allowFullScreen
+        allow="clipboard-write">
+    </iframe>
+</div>
+<p></p>
+
+## Types of Bottom Sheet action
+
+Below are the types of Bottom Sheet actions:
+
+1. **Show**: This opens the bottom sheet.
+2. **Dismiss**: This closes the bottom sheet.
+
+## Opening Bottom Sheet
+
+Follow the steps below to add an action that opens the bottom sheet:
+
+1. Create a bottom sheet [component](../../resources/ui/components/overview.md). Tip: you can also create one from the 'BottomSheet' templates.
```

Comment:
Create this tip into an admonition. And then also link "create components from templates" section under components.